Std/Tactic/TryThis.lean

```diff
@@ -20,28 +20,116 @@ namespace Std.Tactic.TryThis
 
 open Lean Elab PrettyPrinter Meta Server RequestM
 
+/-! # Raw widgets and code actions -/
+
 /--
-This is a widget which is placed by `TryThis.addSuggestion`; it says `Try this: <replacement>`
+This is a widget which is placed by `TryThis.addSuggestion` and `TryThis.addSuggestions`.
+
+When placed by `addSuggestion`, it says `Try this: <replacement>`
 where `<replacement>` is a link which will perform the replacement.
+
+When placed by `addSuggestions`, it says:
+```
+Try these:
+```
+* `<replacement1>`
+* `<replacement2>`
+* `<replacement3>`
+* ...
+
+where `<replacement*>` is a link which will perform the replacement.
 -/
 @[widget] def tryThisWidget : Widget.UserWidgetDefinition where
-  name := "Tactic replacement"
+  name := "Suggestion"
   javascript := "
 import * as React from 'react';
 import { EditorContext } from '@leanprover/infoview';
 const e = React.createElement;
 export default function(props) {
   const editorConnection = React.useContext(EditorContext)
-  function onClick() {
-    editorConnection.api.applyEdit({
-      changes: { [props.pos.uri]: [{ range: props.range, newText: props.suggestion }] }
-    })
+
+  // Construct the style from the object corresponding to a given SuggestionStyle.
+  function getStyle(style) {
```

Comment:
This is a lot of JS code. Can we upstream this to the vscode extension? (Or just not do this...)